yarn.lock

```diff
@@ -17122,9 +17220,9 @@ typescript@4.1.5:
   integrity sha512-6OSu9PTIzmn9TCDiovULTnET6BgXtDYL4Gg4szY+cGsc3JP1dQL8qvE8kShTRx1NIw4Q9IBHlwODjkjWEtMUyA==
 
 typescript@^4.0.2, typescript@^4.0.3, typescript@^4.2.4:
-  version "4.3.5"
-  resolved "https://registry.yarnpkg.com/typescript/-/typescript-4.3.5.tgz#4d1c37cc16e893973c45a06886b7113234f119f4"
-  integrity sha512-DqQgihaQ9cUrskJo9kIyW/+g0Vxsk8cDtZ52a3NGh0YNTfpUSArXSohyUGnvbPazEPLu398C0UxmKSOrPumUzA==
+  version "4.4.2"
```

Comment:
By updating TypeScript it seems like you did not introduce a replaecment for `MediaStreamError`.

See here: https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1029

Please do that replacement.

Reply:
@FlorianRappl the error comes up in the ngx-webcam(in the `node_modules` folder), I went for the following things:

1. Finding a replacement for `MediaStreamError`, which I was not able to find. I found `MediaRecordErrorEvent` which is newly added, but can't say that it's the correct one.
2. Since this error comes in the `ngx-webcam` package, I thought of upgrading it from 0.2.2 to 0.3.x, thinking there might be changes related to this error, but it was not the way I thought it would be.

What else do you think I can do to tackle this?

Right now, I am thinking to revert back to Typescript 4.3 instead of 4.4, what do you think?

Reply:
If its only in a lib you can just disable lib checking.

https://stackoverflow.com/questions/40262513/how-to-disable-suppress-errors-from-libraries-in-typescript

Reply:
Great! That worked!
I didn't know about muting the libraries, before!
Thanks!

Reply:
@FlorianRappl , just last question, should I just disable lib checking in the `esm-form-entry`(using ngx-webcam) or should I put it in the global tsconfig.json?
I think it should be in `esm-form-entry`'s tsconfig instead of global, just wanted to confirm beforehand.

Reply:
Yes, let's keep it as minimal as possible. Only esm-form-entry for now.